### PR TITLE
set the default account upon successful /sync/get call

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

**Comment by kauffj** [comment]
@seanyesmunt this looks good to me, can you review/merge?

File: src/redux/actions/sync.js [hunk 1]
```diff
@@ -31,6 +31,36 @@ export function doSetSync(oldHash, newHash, data) {
   };
 }
 
+export function doSetDefaultAccount() {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.SET_DEFAULT_ACCOUNT,
```
Comment (seanyesmunt) [inline src/redux/actions/sync.js]: Nothing is listening for this? Do we need it?

File: src/redux/actions/sync.js [hunk 2]
```diff
@@ -31,6 +31,36 @@ export function doSetSync(oldHash, newHash, data) {
...
+      }
+
+      // In a case where there's no balance on either account
+      // assume the second (which is created after sync) as default
+      if (!defaultId && accounts.length > 1) {
+        defaultId = accounts[1].id;
+      }
+
+      // Set the default account
+      if (defaultId) {
```
Comment (seanyesmunt) [inline src/redux/actions/sync.js]: If there is no `defaultId`, should we error to say that sync won't work for whatever reason?

File: src/redux/actions/sync.js (hunk 1 above)
Comment (akinwale) [reply src/redux/actions/sync.js]: Just added this in case we may have to use it in the future.

File: src/redux/actions/sync.js (hunk 2 above)
Comment (akinwale) [reply src/redux/actions/sync.js]: No need to throw an error here. If no default id is found, then the wallet will end up being set to the pre-existing default account when it was created.

**Pull request merged**